.github/workflows/ci.yml

```
@@ -223,7 +223,7 @@ jobs:
         uses: ammaraskar/gcc-problem-matcher@master
       - name: Build OpenRCT2
         run: |
-          HOMEBREW_NO_ANALYTICS=1 brew install ninja pkg-config
+          HOMEBREW_NO_ANALYTICS=1 brew install ninja
```

Can we have a comment listed here for those who follow CI instructions?

---

Something like this?
```suggestion
          # NB: GitHub comes with `pkg-config` preinstalled on macOS images
          HOMEBREW_NO_ANALYTICS=1 brew install ninja
```